howso/utilities/feature_attributes/relational.py

```diff
@@ -804,6 +804,24 @@ def _infer_feature_bounds(self,  # noqa: C901
             column_type, _ = self._parse_column_type(str(column.type))
             format_dt = None
 
+            # Compute time-only feature bounds
+            if feature_attributes[feature_name]['data_type'] == 'formatted_time':
+                if (
+                    tight_bounds is None
+                    or feature_name not in tight_bounds
+                ):
+                    if not feature_attributes[feature_name].get('cycle_length'):
+                        raise ValueError(f'Error computing loose bounds for {feature_name}: '
+                                         '`cycle_length` must be specified in attributes')
+                    return {'min': 0, 'max': feature_attributes[feature_name]['cycle_length'],
+                            'allow_null': allow_null}
+                else:
+                    # Expected to return a datetime.time object
+                    min_time, max_time = (
+                        self._get_min_max_values(feature_name))
```

Comment:
Might be good to check the column_type is in `all_date_time_types` like we do for datetimes, othwerwise you might not get the correct values here.